### PR TITLE
Update internal API for ruby 2.6

### PR DESCRIPTION
**Pull request by dalehamel**
@osyo-manga the latest build is broken on ruby 2.6 and above, possibly 2.7 - haven't checked.

Can you please merge this and release a new gem, 0.5.0?

I did a "rake build" and it worked locally

### CODE REVIEW

**Comment by osyo-manga** [comment]
Thank for pull request :)
Also add support for `2.6.0` in [`.travis.yml`](https://github.com/osyo-manga/gem-unmixer/blob/39162c6d39c528cb4a787909bddffbb7317e8600/.travis.yml).

**Comment by dalehamel** [comment]
wow fast turnaround! and woot for adding travis :)

**Comment by dalehamel** [comment]
Ah, compile was not called automatically, it doesn't work after all, getting:

```

In file included from ../../../../ext/unmixer/unmixer.c:9:
../../../../ext/unmixer/2.6.0/internal.h:743:47: error: ‘RVALUE_EMBED_LEN_MAX’ undeclared here (not in a function); did you mean ‘RARRAY_EMBED_LEN_MAX’?
 #   define BIGNUM_EMBED_LEN_MAX (SIZEOF_VALUE*RVALUE_EMBED_LEN_MAX/SIZEOF_ACTUAL_BDIGIT)
                                               ^~~~~~~~~~~~~~~~~~~~
../../../../ext/unmixer/2.6.0/internal.h:756:20: note: in expansion of macro ‘BIGNUM_EMBED_LEN_MAX’
         BDIGIT ary[BIGNUM_EMBED_LEN_MAX];

```

Will have to see where that is defined.

**Comment by dalehamel** [comment]
I couldn't seem to pull in the definition by including <ruby/ruby.h>, so I pulled the enum type into its own "stub" include file. It's a bit messy but it's simple and it works.

looks like it passes travis now @osyo-manga, except for some sporadic errors installing bundler on some ruby versions

File: ext/unmixer/2.6.0/stubs.h [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#ifndef __UNMIXER_RUBY_STUBS__
+#define __UNMIXER_RUBY_STUBS__
```
Comment (osyo-manga) [inline ext/unmixer/2.6.0/stubs.h]: Please change `__UNMIXER_RUBY_STUBS__` to `UNMIXER_RUBY_STUBS`.
`__UNMIXER_RUBY_STUBS__` is reserved names.

Comment (dalehamel) [reply ext/unmixer/2.6.0/stubs.h]: done :)

**Comment by dalehamel** [comment]
@osyo-manga I can't figure out why it is failing with this bundler error, any ideas?

**Comment by osyo-manga** [comment]
Thank!
I fixed it https://github.com/osyo-manga/gem-unmixer/pull/2
Please rebase master.

**Comment by dalehamel** [comment]
@osyo-manga thanks! I pulled in your changes which seem to fix CI, looks all green now.

Anything else holding up a merge?

**Comment by osyo-manga** [comment]
Thanks!
OK, merged.

**Comment by osyo-manga** [comment]
@dalehamel Thank for pull request. Released 0.5.0 https://rubygems.org/gems/unmixer/versions/0.5.0

**Comment by dalehamel** [comment]
thanks @osyo-manga !

**Pull request merged**